packages/apollo-engine-reporting/src/agent.ts

```diff
@@ -122,7 +133,7 @@ export interface EngineReportingOptions<TContext> {
   /**
    * The URL of the Engine report ingress server.
    */
-  endpointUrl?: string;
+  metricsEndpointUrl?: string;
```

Comment:
This is a breaking change, yes? What does that mean for the release?
For anyone that might be overriding this url, if they upgrade their code won't compile? We override this value, for example. 
/cc @trevor-scheer

Reply:
Agreed.